packages/remix-forms/package.json

```diff
@@ -28,9 +28,9 @@
   ],
   "author": "Daniel Weinmann",
   "scripts": {
-    "build": "tsup ./src/index.ts --format esm,cjs --dts --external react",
```

Comment:
This seems to be unnecessary as react is a peerDependency and the `dist/` does not seem to change when I remove it.